sys/src/regress/apexnotecont.c

```diff
@@ -28,12 +28,19 @@ _notetramp(int sig, void (*hdlr)(int, char*, struct Ureg*), struct Ureg *u)
 	if(nstack >= MAXSIGSTACK)
 		noted(1);	/* nesting too deep; just do system default */
 	p = &pcstack[nstack];
+print("a\n");
 	p->restorepc = u->ip;
+print("b\n");
 	p->sig = sig;
+print("c\n");
 	p->hdlr = hdlr;
+print("d\n");
 	p->u = u;
+print("e\n");
 	nstack++;
+print("f\n");
```

Comment:
Are all of these print()'s still necessary? This feels like debugging code that leaked into a commit.